README.md

```diff
@@ -57,27 +57,16 @@ The `docker compose` deployment currently includes four containers
 implementing a fleet of metadata validation web services which are referenced
 inside the testbed as the following named *endpoints*:
 
-- `v09x` represents the current production deployment. Its name reflects
-  that environment's composition: v0.9 of the Shibboleth Metadata
-  Aggregator, and the Xalan XSL processor included in the classpath.
-- `v09` is identical to `v09x` with the exclusion of the Xalan processor.
-  Because this can run neither the old nor new versions of the URL checking
-  code (the old system involves a Xalan extension, while the new system only
-  exists under MDA v0.10), we do not expect to ever use this configuration
-  in production.
-- `v010x` is anticipated to represent the next step in the evolution of
-  the UK federation tooling, switching out MDA v0.9 for MDA v0.10 but
-  retaining Xalan for compatibility until we are certain that all of the
-  XSL in the production deployment behaves the same under the XSL processor
-  included in the JDK.
-- `v010` is, for now, the anticipated final state for the UK federation
-  deployment once all compatibility issues are resolved so that Xalan can
-  be removed.
+- `v010` represents the current production environment: v0.10 of
+  the Shibboleth Metadata Aggregator, *without* the use of Xalan
+  and extensions.
+- `v010x` is ther same as `v010` but with the use of Xalan and
```

Comment:
'ther' -> 'the'

Reply:
I think that's a transient error as the final version of `README.md` replaces that text a second time. Can you check the state in the last commit in the PR to confirm?

Reply:
You are quite right. It is 'next' in the final readme without a typo.